config.yml

```
@@ -1,20 +1,37 @@
 title: GitHub Pages
 tagline: Your sidekick in using GitHub Pages and blogs
-description: Learn how to create a site or blog from your GitHub repositories with GitHub Pages.
+description: >-
+  Learn how to create a site or blog from your GitHub repositories with GitHub
+  Pages.
 tags:
   - GitHub
   - GitHub Pages
 template:
   name: github-pages-with-jekyll
   repo: github-pages-with-jekyll-template
+preferences:
+  - type: dropdown
+    name: language
+    label: Language
+    description: Choose your preferred language.
+    options:
+      - label: English
+        value: en
+      - label: Español
+        value: es
 before:
   - type: createIssue
     title: Welcome
     body: 00_class-introduction-issue.md
-    
 steps:
   - title: Generate a GitHub Pages site
-    description: Generate a GitHub Pages site with the automatic page generator. 
+    description: Generate a GitHub Pages site with the automatic page generator.
+    translations:
+      es:
+        title: Activa GitHub Pages
+        description: >-
+          Gnera un sitio web con el generador automático de páginas GitHub
```

```suggestion
          Genera un sitio con el generador automático de GitHub Pages
```
Please be patient with me since I am not sure how to review translations well. 

I am not so sure about this translation. The title states `Activate GitHub Pages` however the description says `Generate a website with the automatic generator from GitHub Pages`. It's unclear to me if we are activating or generating a site. 

Another thing I am not sure about is translating product names. It looks like in one case we refer to the product by its English name and then in the description we translated it?  `páginas GitHub` vs `GitHub Pages`. Not sure if we want to translate those or not 🤷‍♂